src/plugins/chessevent/chessevent_session.py

```diff
@@ -0,0 +1,89 @@
+from logging import Logger
+
+from requests import Session, Response
+from requests.exceptions import RequestException
+
+from common import SharlyChessException
+from common.exception import ImporterError
+from common.i18n import _
+from common.logger import get_logger
+
+logger: Logger = get_logger()
+
+
+class ChessEventSession(Session):
+    """A Requests session specialised for communication with
+    the ChessEvent platform."""
+
+    DOWNLOAD_URL: str = 'https://chessevent.echecs-bretagne.fr/download'
+
+    def __init__(
+        self, user_id: str, password: str, event_id: str, tournament_name: str
+    ):
+        super().__init__()
+        self.user_id = user_id
+        self.password = password
+        self.event_id = event_id
+        self.tournament_name = tournament_name
+
+    def read_data(self) -> str:
+        """Reads the data of a ChessEvent tournament."""
+        post: dict[str, str] = {
+            'user_id': self.user_id,
+            'password': self.password,
+            'event_id': self.event_id,
+            'tournament_name': self.tournament_name,
+        }
+        logger.debug(
+            'Reading data from the ChessEvent platform (%s)...',
+            f'{self.user_id}:{"*" * 8}@{self.event_id}/{self.tournament_name}',
```

Comment:
As long as the tournement name is not an ID (any string).
```suggestion
            f'{self.user_id}:{"*" * 8}@{self.event_id}/[{self.tournament_name}]',
```